pyk/src/pyk/kore/rpc.py

```diff
@@ -341,23 +341,24 @@ def close(self) -> None:
 
     def request(self, method: str, **params: Any) -> dict[str, Any]:
         old_id = self._req_id
+        req_id = f'{id(self)}_{old_id:03}'
```

Comment:
Nitpick: can we make `-` the separator?

```suggestion
        req_id = f'{id(self)}-{old_id:03}'
```